randovania/games/fusion/logic_database/header.txt

```diff
@@ -86,7 +86,7 @@ Templates
 * 10 DMG Missiles:
       All of the following:
           Missiles
-          Ice Missile Data or Missile Data
+          Ice Missile Data or Missile Data or Super Missile Data
```

Comment:
super missiles do 20 damage and shouldn't be in here

Reply:
20 > 10

Reply:
to clarify: yes, they absolutely should be in here. we have in multiple places requirements with something like "10 missiles & 10 dmg or 1 missile & 45 dmg". A super missile would, because 20 is greater than 10, fullfill the first condition. but if we never list supers, then they never will be logically considered.
in other words, our templates are "deal at least x dmg" not "deal exactly x dmg"

Reply:
I would like the template to be renamed to clarify this, something like `10+ DMG Missiles` perhaps?

Reply:
sure

Reply:
only renamed this one, as 20, 30 and 45 are always exact